src/shapes/IText/ITextBehavior.ts

```diff
@@ -150,7 +150,7 @@ export abstract class ITextBehavior<
 
   private _tick(delay?: number) {
     this._currentTickState = this._animateCursor({
-      toValue: 1,
+      toValue: 0,
       duration: this.cursorDuration,
       delay,
       onComplete: this._onTickComplete,
```

Comment:
if this becomes the disappearing tick, because of the discussion on the PR and also because of let's try a change at time, duration here should be this.cursorDuration / 2, and delay should be minimum 100 or the number passed to as parameter.